packages/edit-site/src/components/block-editor/style.scss

```diff
@@ -158,3 +158,7 @@
 		box-shadow: inset 0 0 0 2px var(--wp-admin-theme-color);
 	}
 }
+
+.edit-site-visual-editor__editor-canvas.is-canvas-view {
+	cursor: pointer;
```

Comment:
For some reason, I don't see the cursor changing, I'm not sure I understand the reason. @jasmussen any idea?